docker-compose.yml

```diff
@@ -41,6 +41,18 @@ services:
       env_file:
         - .env
 
+    osrm:
+        build:
+          context: ./osrm
+          dockerfile: Dockerfile
+        volumes:
+          - ./data/:/data/:rw
+          - ./osrm/profiles/car-shortest-distance.lua:/opt/car-shortest-distance.lua
```

Comment:
You can simply load the folder, so that if we add lua scripts, they will get loaded as well

Reply:
What's the syntax for this? When I tried initially it looked like the entire `/opt/` folder had been replaced.

Reply:
Hmm, not sure, maybe add a /profile/ sub-level on the instance?